### PR TITLE
Development

### PR DESCRIPTION
**Pull request by alfonsotech**

### CODE REVIEW

**Comment by alfonsotech** [comment]
I wrote all the func into the app.js file and am in the process of giving it a better structure.

File: app.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+'use strict'
+
+const express = require('express')
+const app = express()
+const Twit = require('twit')
+const config = require('./config.js')
+const T = new Twit(config)
```
Comment (FrankieFabuloso) [inline app.js]: I would rename this variable to be a little more meaningful. I still don't really know what this `T` object is.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+//MODEL
+// const allTweets = [];
+
+function getAll() {
+	database.getAllTweets().then(data => {
+		for(var i = 0; i < data.length; i++) {
+			allTweets.push(data[i].tweetx)
+		}
+
+		// console.log('allTweets', allTweets)
```
Comment (FrankieFabuloso) [inline app.js]: remove all comments and console.logs. It makes it less distracting to give meaningful reviews if I don't have to process filtering all the comments and console.logs from your code as well as trying to understand what it is doing.

File: app.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+	})	
+}
+function getOne() {
+	getAll()
+	console.log('allTweets', allTweets)
+	const arrayLength = allTweets.length
+	console.log('arrayLength', arrayLength)
+	let index = Math.floor((Math.random() * arrayLength) + 1);
+	console.log('index, tweet at index: ', index, allTweets[index])
```
Comment (FrankieFabuloso) [inline app.js]: remove console.logs before submitting a PRR, please.

Add newline between variable declarations and body of your function.

Try to not use let, unless you have to. Doesn't seem like you need index to be mutable here.

```
const arrayLength = allTweets.length
const  index = Math.floor((Math.random() * arrayLength) + 1);

return allTweets[index]
```

File: app.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+	q: 'tPhilosophia',
+	count: 50
+}
+
+T.get('search/tweets', params, getTweets)
+
+function getTweets(err, data, response) {
+	var tweets = data.statuses
+	for(var i = 0; i < tweets.length; i++) {
+		// let test = tweets[i].text
```
Comment (FrankieFabuloso) [inline app.js]: remove comments before submitting a PRRR

File: app.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+		database.addTweet(tweets[i].text)
+	}
+}
+
+//	TWEET OUT ONE RANDOM TWEET FROM DB
+setInterval (tweetOut, 1000*60*60)
```
Comment (FrankieFabuloso) [inline app.js]: Why are you passing in an expression rather than just a value?

600000 reads better than 1000\*60\*60.

File: app.js [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+function tweetOut() {
+	const allTweets = [];
+	database.getAllTweets().then(data => {
```
Comment (FrankieFabuloso) [inline app.js]: I prefer chaining a `.then` on a new line:

```
database.getAllTweets()
.then( data => { ... } )

File: app.js [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+		for(var i = 0; i < data.length; i++) {
```
Comment (FrankieFabuloso) [inline app.js]: don't use var, let and const give you better scope control and are more meaningful in terms of code clarity.

File: app.js [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+			allTweets.push(data[i].tweetx)
+		}
+		let arrayLength = allTweets.length
+		let index = Math.floor((Math.random() * arrayLength) + 1);
```
Comment (FrankieFabuloso) [inline app.js]: These lets can probably be const, it doesn't seem like you are mutating them anywhere in the function.

**Pull request merged**